src/fides/api/models/privacy_request.py

```diff
@@ -660,6 +660,24 @@ def get_cached_access_graph(self) -> Optional[GraphRepr]:
         ] = cache.get_encoded_objects_by_prefix(f"ACCESS_GRAPH__{self.id}")
         return list(value_dict.values())[0] if value_dict else None
 
+    def cache_data_use_map(self, value: Dict[str, Set[str]]) -> None:
+        """
+        Cache a dict of collections traversed in the privacy request
+        mapped to their associated data uses
+        """
+        cache: FidesopsRedis = get_cache()
+        cache.set_encoded_object(f"DATA_USE_MAP__{self.id}", value)
```

Comment:
We also have a .fides/redis_dataset.yml that stores the fields in our redis cache that we can add to here - there's nothing that automatically checks this, it was just something I was asked to make when we were merging ctl and ops.

Reply:
got it, thank you for calling that out! i wonder if/how we could document that in the codebase 🤔